.github/ISSUE_TEMPLATE/bug_report.md

```
@@ -2,39 +2,38 @@
-- OS: [e.g. iOS8.1]
-- Browser [e.g. stock browser, safari]
-- Version [e.g. 22]
-
-**Additional context**
-Add any other context about the problem here.
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: textarea
```

Markdownlint (0.37.0)

Expected: 0; Actual: 2
Unordered list indentation

(MD007, ul-indent)